### PR TITLE
195 make courses searchable

### LINKED ISSUE #195: Make courses searchable
As a user, I want to find information about courses at the HPI quickly.

To be able to access this information, the detail page for the course should be appearing when I search for the course name.

### PR DESCRIPTION
**Pull request by lwenner**
Courses are now searchable.

closes #195

### CODE REVIEW

File: app/models/course.rb [hunk 1]
```diff
@@ -1,7 +1,24 @@
...
+# The model representing a course held at HPI
+class Course < SearchableRecord
   validates :name, presence: true
 
   has_and_belongs_to_many :people
   has_many :course_times, dependent: :destroy
   belongs_to :room, optional: true
+
+  def image_or_placeholder
+    "https://via.placeholder.com/512?text=Course"
```
Comment (Margarete01) [inline app/models/course.rb:10]: There should be a placeholder_course.png in the assets which should be used here

File: spec/factories/people.rb [hunk 2]
```diff
@@ -5,6 +5,7 @@
     first_name { "Michael" }
     title { "Dr." }
     status { "Chair Representative" }
+    informations { [ Information.create(key: 'telegram', value: '@perscheid') ]}
```
Comment (Margarete01) [inline spec/factories/people.rb:8]: Why did you add this line of code?

Comment (lwenner) [reply spec/factories/people.rb:8]: We have added tests to check if there are bagdes when searching for people. Before that, there was no information about a person and, accordingly, no badges.

**Review by cdfhalle — APPROVED** [review]
schon nice
